builtin/logical/rabbitmq/path_config_connection.go

```diff
@@ -91,6 +100,33 @@ func (b *backend) pathConnectionUpdate(ctx context.Context, req *logical.Request
 	return nil, nil
 }
 
+func readConfig(ctx context.Context, storage logical.Storage) (config connectionConfig, err error) {
+	entry, err := storage.Get(ctx, storageKey)
+	if err != nil {
+		return connectionConfig{}, err
+	}
+	if entry == nil {
+		return connectionConfig{}, nil
+	}
+
+	var connConfig connectionConfig
+	if err := entry.DecodeJSON(&connConfig); err != nil {
+		return connectionConfig{}, err
+	}
+	return connConfig, nil
+}
+
+func writeConfig(ctx context.Context, storage logical.Storage, config connectionConfig) (err error) {
```

Comment:
nit: It doesn't seem like the named return `err` is being used in this function since a new `err` is declared every time it's used.

```suggestion
func writeConfig(ctx context.Context, storage logical.Storage, config connectionConfig) error {
```